frontend/src/i18n/config.ts

```diff
@@ -4,6 +4,25 @@ import { initReactI18next } from 'react-i18next';
 
 const en = {}; // To keep TS happy.
 
+export const supportedLanguagesWithLabels = [
+  {
+    lng: 'en',
```

Comment:
Let's call this `id` or `locale` instead of `lng`, or you can just reuse the array of the language IDs with the `Intl` method I mentioned.

Reply:
fixed